pages/admin/request/[requestId].tsx

```diff
@@ -0,0 +1,219 @@
+import { GetServerSideProps } from 'next'; // Get server side props
+import { getSession } from 'next-auth/client'; // Session management
+import { useState } from 'react'; // React
+import { GridItem, Stack } from '@chakra-ui/react'; // Chakra UI
+import Layout from '@components/internal/Layout'; // Layout component
+import { Role } from '@lib/types'; // Role enum
+import { authorize } from '@tools/authorization'; // Page authorization
+
+import RequestHeader from '@components/requests/RequestHeader'; // Request header
+import DoctorInformationCard from '@components/requests/DoctorInformationCard'; // Doctor information card
+import PaymentInformationCard from '@components/requests/PaymentInformationCard'; // Payment information card
+import PersonalInformationCard from '@components/requests/PersonalInformationCard'; // Personal information card
+import ReasonForReplacementCard from '@components/requests/ReasonForReplacementCard'; // Reason for replacement card
+import ProcessingTasksCard from '@components/requests/ProcessingTasksCard'; // Processing tasks card
+
+// TEMPORARY MOCK DATA
+
+// TODO: REMOVE THIS LATER. Need this to pass pipeline b/c mock applicant needs to satisfy the real Applicant type.
+/* eslint-disable @typescript-eslint/no-empty-function */
+import { Gender, Province, PhysicianStatus, PaymentType } from '@lib/types';
+
+type ApplicationStatus =
+  | 'COMPLETED'
+  | 'INPROGRESS'
+  | 'PENDING'
+  | 'REJECTED'
+  | 'EXPIRING'
+  | 'EXPIRED'
+  | 'ACTIVE';
+const mockApplication = {
```

Comment:
When you work on API Hookups, it would be really nice if we could keep this mock data and use it in our seeding script!